identity.go

```diff
@@ -76,9 +76,15 @@ type Identity interface {
 	// StopWatchingFiles reversed WatchFiles.
 	StopWatchingFiles()
 
+	// IsCertSettable returns nil if the "cert" certificate storage supports writing, used before calling SetCert()
+	IsCertSettable() error
```

Comment:
imo, it seems strange for an 'Is' function to return error not a bool

Reply:
:D if you say so. I'd just prefer the "Is" to be "Verify" or "Check" or literally anything that isn't "Is" but i don't care all that much